<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZWebDowloadManager</class>
 <widget class="QDialog" name="ZWebDowloadManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>397</width>
    <height>274</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#ZWebDowloadManager{border-image: url(:/new/imageFile/ZWebViewBrowserImage/downloadList.png);background-color: rgb(255, 255, 255);}

</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>475</x>
     <y>5</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image: url(:/new/imageFile/ZWebViewBrowserImage/close_nor.png);}
QPushButton:hover{border-image: url(:/new/imageFile/ZWebViewBrowserImage/close_sel.png);}
QPushButton:pressed{border-image: url(:/new/imageFile/ZWebViewBrowserImage/close_sel.png);};</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="EditTableView" name="downloadsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>38</y>
     <width>499</width>
     <height>251</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>390</width>
     <height>200</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">gridline-color: black;        /*表格内框颜色*/
border-color: rgb(212, 212, 212);
</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>481</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font: 14px &quot;微软雅黑&quot;;color: rgb(102, 102, 102)</string>
      </property>
      <property name="text">
       <string>下载数</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="itemCount">
      <property name="styleSheet">
       <string notr="true">font: 12px &quot;微软雅黑&quot;;color: rgb(102, 102, 102)</string>
      </property>
      <property name="text">
       <string>0 Item</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="cleanupButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>79</width>
        <height>20</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>79</width>
        <height>20</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{border-image: url(:/new/imageFile/ZWebViewBrowserImage/clearReload.png);}
QPushButton:hover{border-image: url(:/new/imageFile/ZWebViewBrowserImage/clearReload_click.png);}
QPushButton:pressed{border-image: url(:/new/imageFile/ZWebViewBrowserImage/clearReload_click.png);};</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EditTableView</class>
   <extends>QTableView</extends>
   <header location="global">ZWebViewBrowser/edittableview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ZWebDowloadManager</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
